### ISSUE TITLE
yumrepo listed in Ansible module packaging documentation, but not shipped in epel package

### ISSUE THREAD

**Issue by ahuffman** [ansible/ansible-modules-extras#2063]
##### ISSUE TYPE
- Bug Report
- Documentation Report
##### COMPONENT NAME

yumrepo
##### ANSIBLE VERSION

```
ansible 2.0.1.0
```
##### CONFIGURATION

forks=50
scp_if_ssh = True
##### OS / ENVIRONMENT

RHEL7
##### SUMMARY

Trying to use the yumrepo module in a role, but getting error: 
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

This corresponds to the task trying to make use of the yumrepo module.
##### STEPS TO REPRODUCE

```
- name: Setup CentOS7-Extras Repository for cockpit-kubernetes
  yum_repository:
    file: centos7-extras.repo
    name: centos7-extras
    description: CentOS7 - Extras
    baseurl: http://mirror.centos.org/centos/7/extras/x86_64/
    enabled: yes
    gpgcheck: yes
    gpgkey: http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7
    includepkgs: cockpit-kubernetes
  when: k8s_cockpit == 'yes'
```
##### EXPECTED RESULTS

A yumrepo would be created on the selected host.
##### ACTUAL RESULTS

ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/etc/ansible/roles/k8s-master/tasks/main.yml': line 12, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:
- name: 'Setup CentOS7-Extras Repository for cockpit-kubernetes'
  ^ here

**Comment by Collin-Molnar** [comment]
I am also seeing this issue, installed ansible via pip, using version 2.0.1.0. Note on the documentation page it looks like this is available in versions >= 2.1. Looks like 2.1 isn't around yet.

**Comment by bcoca** [comment]
to list installed available modules use `ansible-doc -l`, the docsite reflects 'lastest' docs which might include modules you don't have or have not shipped yet. Please see the 'version added' info.

Closing this as the module is not supposed to be in 2.0.1

**Comment by Collin-Molnar** [comment]
@bcoca,

When will version 2.1 be available? That's the version the docs state that yum_repository will be shipped with ansible, but the latest version available via pip is 2.0.0.2.

**Comment by bcoca** [comment]
end of April, latest stable should be 2.0.2, which is 2 versions AFTER 2.0.0.2
